build-support/bin/check_pants_pex_abi.py

```diff
@@ -32,7 +29,7 @@ def main() -> None:
   if not parsed_abis.issubset(expected_abis):
     die("pants.pex was built with the incorrect ABI. Expected wheels with: {}, found: {}."
         .format(' or '.join(sorted(expected_abis)), ', '.join(sorted(parsed_abis))))
-  success("Success. The pants.pex was built with wheels carrying the expected ABIs: {}."
+  green("Success. The pants.pex was built with wheels carrying the expected ABIs: {}."
```

Comment:
This might be super bike-sheddy, but I liked the extra semantics conveyed by this being `success`, meaning that we don't care about which colour this is printed in, we just have to notify success. I'd keep the `success` function below, and just call `green` from it, since imo `main` shouldn't care about which colour means "success". *However*, this is a super-nit, and in a script this size it might not even be worth it, so feel free to ignore it.

Reply:
Or just rename `green` to `success`, since `die` exists.

Reply:
I see what you mean and agree, although for consistency with `common.sh` am going to keep it as `green`. I want those APIs as similar as possible to make it really easy to develop either a bash script or a Python script.